src/client/componentsMealsharing/AllReservations.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "../App.css";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+function AllReservations() {
+  const [fetchData, setFetchData] = useState([]);
+
+  useEffect(() => {
+    fetchDataResult();
+  }, []);
+
+  const fetchDataResult = async () => {
+    const response = await fetch("http://localhost:3000/api/reservations");
+    const Data = await response.json();
+    setFetchData(Data);
```

Comment:
camelCase variables :) 
```suggestion
    const data = await response.json();
    setFetchData(data);
```